frontend/src/pages/GranteeRecord/index.js

```diff
@@ -24,7 +26,7 @@ export default function GranteeRecord({ match, location }) {
     granteeId,
   });
 
-  const defaultFilters = [
+  const [defaultFilters] = useState([
     {
```

Comment:
this is weird

Reply:
Yea I think this could be a common problem for filters, having set filters that aren't exposed to the UI and cannot change. I wonder if there is a better way to handle it?

Reply:
Looked at it with fresh eyes, made some (hopefully) better choices